### ISSUE TITLE
Loading of draco file fails

### ISSUE THREAD

**Issue by floAr** [atteneder/glTFast#50]
I'm getting a decoding error when trying to load draco compressed files. 
```
Failed: Decoding error.
UnityEngine.Debug:LogError(Object)
GLTFast.PrimitiveDracoCreateContext:CreatePrimitive() (at Library/PackageCache/com.atteneder.gltfast@64f3c1b978d7139af3da6d9c48b02ae06a375d90/Runtime/Scripts/GltFast.cs:159)
GLTFast.<Prepare>d__43:MoveNext() (at Library/PackageCache/com.atteneder.gltfast@64f3c1b978d7139af3da6d9c48b02ae06a375d90/Runtime/Scripts/GltFast.cs:545)
GLTFast.<LoadContent>d__8:MoveNext() (at Library/PackageCache/com.atteneder.gltfast@64f3c1b978d7139af3da6d9c48b02ae06a375d90/Runtime/Scripts/GltfAsset.cs:77)
UnityEngine.SetupCoroutine:InvokeMoveNext(IEnumerator, IntPtr)
```

I tried running with 2019.2 as well as 2019.3 on windows 10, git as well as lsf are up to date.
Packages appear to also be in order and up to date:
```
{
  "dependencies": {
    "com.atteneder.draco": "https://gitlab.com/atteneder/DracoUnity.git",
    "com.atteneder.gltfast": "https://github.com/atteneder/glTFast.git",
    "com.unity.ext.nunit": "1.0.0",
    "com.unity.ide.vscode": "1.1.2",
    "com.unity.test-framework": "1.1.1",
    "com.unity.modules.imageconversion": "1.0.0",
    "com.unity.modules.imgui": "1.0.0",
    "com.unity.modules.jsonserialize": "1.0.0",
    "com.unity.modules.unitywebrequest": "1.0.0",
    "com.unity.modules.unitywebrequesttexture": "1.0.0"
  },
  "testables": [
    "com.atteneder.gltfast"
  ],
  "lock": {
    "com.atteneder.draco": {
      "revision": "HEAD",
      "hash": "b04c2d09ff4d4889e68e0cdfe6b3dc47f4a58a29"
    },
    "com.atteneder.gltfast": {
      "revision": "HEAD",
      "hash": "64f3c1b978d7139af3da6d9c48b02ae06a375d90"
    }
  }
}
```

I created the file using the [fbx2gltf ](https://github.com/facebookincubator/FBX2glTF/releases) converter 0.9.7 using .\FBX2glTF-windows-x64.exe --input .\Y_Bot.fbx --output Y_Bot.glb --binary --draco (also tried without --binary)

I attached the file in question and would be extremly happy for any pointers where to start debugging / fixing the whole thing!

[data.zip](https://github.com/atteneder/glTFast/files/3904456/data.zip)

**Comment by atteneder** [comment]
Thanks for the report!
Draco in combination with glTF-binary does not work, because the main buffer access is off. For some reason the default sample models don't contain a file like that :)
Expect a fix today/tomorrow.

**Comment by atteneder** [comment]
Fixed in master and upcoming release 0.7.1
Nice model, by the way.
Thanks again for the report!
